src/Pages/HomePage.tsx

```diff
@@ -15,28 +10,44 @@ import { Header } from '../components/Header'
 import { loggeOut } from '../components/HeaderNavLink/HeaderNavLink'
 import { PageFooter } from '../components/PageFooter/PageFooter'
 import footerLinks from '../utils/FooterLinks'
-import ListOffers, { ListOffersProps } from '../components/ListOffers/ListOffers'
+import ListOffers, {
+  ListOffersProps,
+} from '../components/ListOffers/ListOffers'
 
+import {
+  categoriesSelector,
+  fetchCategories,
+} from '../redux/slices/categoriesSlice'
+import { useAppDispatch } from '../redux/hooks'
+import { useSelector } from 'react-redux'
+import { useEffect } from 'react'
+import { TopCategories } from '../components/TopCategories/TopCategories'
 export const HomePage = () => {
   const dispatch = useAppDispatch()
   const offer = useSelector(adsSelector)
-  
-  const ListOfOffers = function(props: ListOffersProps): JSX.Element {
-    return <ListOffers {...props}/>
+  const ListOfOffers = function (props: ListOffersProps): JSX.Element {
+    return <ListOffers {...props} />
```

Comment:
Dont think we need this here , just use ListOffers directly.

Remove the above